### ISSUE TITLE
button purpose (in context)

### ISSUE THREAD

**Issue by goetsu** [w3c/wcag#168]
Can anyone explain me why wcag are not allowing context for `<button type="button">` outside of form like it do for links at level A?

**Comment by awkawk** [comment]
Links were called out specially in WCAG 2.0, likely because the common use of "read more" and "click here" links resulted in a success criteria that addressed that practice perhaps too narrowly.  It is unfortunate that... 

<h3>Article 1</h3><p>This article is about really interesting things like... <button type="button>Read more</button>

... would fail 4.1.2 but the same pattern using a link that is styled to look exactly the same wouldn't.  I do not believe that this was the intent of the group is well-served by the narrow language.  

While WCAG 2.0 is not focused only on HTML, I think that the use of link in the standard makes it clear that it is talking about hyperlinks as being distinct from buttons.

I propose that we add this as an item to the "post WCAG 2.0" list of topics to address.

**Comment by goetsu** [comment]
Hi, I think the end of your comment is missing.
If I understand we can't change the 2.4.4 itself maybe some note and technique can be added / updated to reflect this

**Comment by awkawk** [comment]
Thanks @goetsu, should be better now.

**Comment by awkawk** [comment]
The WCAG Group met on April 12, 2016 (http://www.w3.org/2016/04/12-wai-wcag-minutes.html) and arrived at the following resolution (to be approved by CfC process):

The Working Group has concluded that WCAG 2.0 SC 2.4.4 does not cover button elements per a strict read of WCAG 2.0. The Working Group resolves to add this as an issue for consideration in a post-WCAG 2.0 version

**Comment by joshueoconnor** [comment]
I discussed this (informally this evening but F2F) with @goetsu and he seemed to be happy for this issue to be dealt with in a WCAG.next. @goetsu is this ok?

**Comment by goetsu** [comment]
yes it's ok for me

**Comment by mraccess77** [comment]
I am fine dealing with this in WCAG Next -- In addition, I would point out that SC 2.4.6 deals with headings and labels that describe the topic or purpose.  It seems the term label could be applied to buttons and if the label did not describe the topic or purpose then this SC could be used.

**Comment by awkawk** [comment]
CfC passed: https://lists.w3.org/Archives/Public/w3c-wai-gl/2016AprJun/0150.html

**Comment by spanchang** [comment]
Button purpose  is covered at Level A and Level AA even today:
Button's purpose has to be clear as per SC 1.1.1: non-text content is a control or accepts user input, then it has a 
name that describes its purpose. 

A button's "label" is also covered by 2.4.6. See definition of label
I agree that SC 2.4.4 does not apply to buttons but only linkks. 
Sailesh

**Comment by goetsu** [comment]
@spanchang I don't see how button are non-text content and accepts user input example <button>Close</button>.
I agree that 2.4.6 cover it but my question was about allowing context like we allow for links because I don't see any reason not doing it for button
